wiki/Tournaments/Official_support/ko.md

```diff
@@ -119,15 +154,15 @@ tags:
 
 지원 팀의 재량에 따라, 토너먼트에서 공식 지원을 잃게 되는 행위를 한 사람은 최소 4개월 동안 배지가 있는 다른 토너먼트에서 동일한 역할을 하거나 호스트 또는 관리자가 될 수 없습니다. 이 제한이 유저에게 적용될 경우, [토너먼트 팀](mailto:tournaments@ppy.sh)을 통해 유저에게 직접 통보됩니다.
 
-모든 토너먼트 스태프의 명단은 쉽게 접근할 수 있는 위치에 공개되어야 합니다. 이 명단은 토너먼트 포럼 게시물, 웹사이트 혹은 스프레드시트에 포함해야 할 수도 있습니다.
+모든 토너먼트 스태프의 명단은 쉽게 접근할 수 있는 위치에 공개되어야 합니다. 이 명단은 토너먼트 포럼 게시물, 위키 페이지, 웹사이트 혹은 스프레드시트에 포함해야 할 수도 있습니다.
```

Comment:
Missing the "If there are multiple lists of staff, they must all be up to date and in sync with one another." change here.